### PR TITLE
Vietnamese translation

### PR DESCRIPTION
**Pull request by IntelCorei5-8350u**
Did some tweaks so the translation makes sense to most Vietnamese while keeping the original meaning. Word-by-word translation was impossible.

### CODE REVIEW

File: tizenbrew-app/TizenBrew/lang/en.json
Comment (reisxd) [inline tizenbrew-app/TizenBrew/lang/en.json:1]: Why was this deleted?

**Review by reisxd — CHANGES_REQUESTED** [review]

File: tizenbrew-app/TizenBrew/lang/en.json
Comment (IntelCorei5-8350u) [reply tizenbrew-app/TizenBrew/lang/en.json:1]: Re added en.json

**Review by reisxd — APPROVED** [review]
